frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -110,8 +174,75 @@ const filters = [
   },
 ];
 
-const NodesPage = (props) => (
-  <ListPage {...props} ListComponent={NodesTable} rowFilters={filters} />
-);
+const fetchNodeMetrics = (): Promise<NodeMetrics> => {
+  const metrics = [
+    {
+      key: 'freeMemory',
+      query: 'node_memory_MemAvailable_bytes',
+    },
+    {
+      key: 'totalMemory',
+      query: 'node_memory_MemTotal_bytes',
+    },
+    {
+      key: 'freeStorage',
+      query: 'node_filesystem_free_bytes',
+    },
+    {
+      key: 'totalStorage',
+      query: 'node_filesystem_avail_bytes',
+    },
+    {
+      key: 'cpu',
+      query: 'instance:node_cpu:rate:sum',
+    },
+  ];
```

Comment:
cc @s-urbaniak @kyoto for review of these queries

Reply:
CC @openshift/openshift-team-monitoring

Reply:
cc @paulfantom as you have more background in node metrics.